devel/npm6/Portfile

```diff
@@ -24,10 +24,9 @@ distname            npm-${version}
 
 extract.suffix      .tgz
 
-checksums           sha1    30ed48d4cd4d17d68ee04a5fcf9fa2ca9167d819 \
```

Comment:
npm publishes SHA-1 checksums (e.g. .versions."6.6.0".dist.shasum) in https://registry.npmjs.org/npm, so please keep it.

Reply:
If there's a good reason to keep the SHA-1 checksum even though it isn't very secure any more, we should put in a comment indicating that so the rationale is documented.